fulp_modules/ipc/code/surgery/body_repair.dm

```diff
@@ -0,0 +1,128 @@
+//Almost copypaste of tend wounds, with some changes
+/datum/surgery/robot_healing
+	steps = list(/datum/surgery_step/mechanic_open,
+				/datum/surgery_step/pry_off_plating,
+				/datum/surgery_step/cut_wires,
+				/datum/surgery_step/robot_heal,
+				/datum/surgery_step/mechanic_close)
+
+	requires_bodypart_type = BODYPART_ROBOTIC
```

Comment:
is it intended that, unlike robotic!organ manipulation, you can't do this surgery on yourself?

Reply:
lol